ipp_macro_series_parser/comptes_nationaux/cn_main.py

```diff
@@ -27,4 +29,16 @@ def cn_df_generator(year, list_years = None):
     return df_full
 
 
-table = cn_df_generator(2013, range(2010, 2014))
+def excel_table_generator(folder_year, entry_by_index_list, output_path):  # to add: argument list_years
+    df = cn_df_generator(folder_year)
+    extractor(df, entry_by_index_list, output_path)
+
+
+# This is a test of most of the methods
+year = 2013
+entry_by_index_list = [{'code': None, 'institution': 'S1', 'ressources': False, 'description': 'PIB'},
+             {'code': None, 'institution': 'S1', 'ressources': False, 'description': 'PIN'}]
+output_path = "table_for_IPP.xlsx"
+
+excel_table_generator(year, entry_by_index_list, output_path)
+os.system('start excel.exe {}'.format(output_path))
```

Comment:
Could you put this example (lines 37 to 44) as a docstring for the function excel_table_generator ?
See cn_extract_data.py for examples of docstrings